core/modules/system/system.menu.inc

```diff
@@ -204,7 +207,15 @@ function system_menu_block_build(array $config) {
       $id = backdrop_html_id('menu-toggle-state');
       $data['content']['#wrapper_attributes']['data-menu-toggle-id'] = $id;
       $data['content']['#prefix'] = theme('menu_toggle', array('enabled' => $config['toggle'], 'id' => $id, 'text' => t('Menu')));
-      $data['content']['#attached']['css'][] = backdrop_get_path('module', 'system') . '/css/menu-toggle.theme.css';
+      if (config_get('system.core', 'menu_breakpoint') == 'default') {
+        $data['content']['#attached']['css'][] = backdrop_get_path('module', 'system') . '/css/menu-toggle.theme.css';
+      }
+      else {
+        $data['content']['#attached']['css'][] = backdrop_get_path('module', 'system') . '/css/menu-toggle.theme.breakpoint.css';
+        $data['content']['#attached']['css'][backdrop_get_path('module', 'system') . '/css/menu-toggle.theme.breakpoint-queries.css'] = array(
+          'media' => 'all and (min-width: ' . config_get('system.core', 'menu_breakpoint_custom') . ')',
```

Comment:
This is smart. I don't think I would have thought of this, but it's instantly clear what it's doing. Nice job :+1: